### ISSUE TITLE
bug: the client closes in the crafting window

### ISSUE THREAD

**Issue by EspectroW94** [AscensionGameDev/Intersect-Engine#2146]
### Description

When you place the mouse pointer over an item in the ingredients, which is an equippable item, the client closes.

I leave a video

https://github.com/AscensionGameDev/Intersect-Engine/assets/127634629/d5d1d25c-edfe-4ad9-abdd-d047f9476d93

### Steps to Reproduce

1. create a recipe that includes an equippable item
2. place the mouse pointer over the ingredient that is an equippable item

### Version with bug

0.8.0.125

### Last version that worked well

0.8.0.111

### Affected platforms

Windows, I was *not* able test on other platforms

### Duplicate Bug Check

- [X] This bug report is not a duplicate to the best of my knowledge.

**Comment by cydyn** [comment]
This is not the only problem. I noticed that if an item has statistics, even moving the mouse to the item to be created also crashes the client.
Version 0.8.121.

**Comment by TimtyG** [comment]
This is duplicate of #2135 , it should get fixed once the new PR is approved and merged.

**Comment by Arufonsu** [comment]
not entirely a duped issue but it was indeed,[ mentioned here before](https://github.com/AscensionGameDev/Intersect-Engine/issues/2135#issuecomment-1855697283)
closing this as dupe of #2135 , which is linked to #2145 as a WIP/Waiting for review
